dotnet/Directory.Packages.props

```diff
@@ -4,6 +4,8 @@
     <MicrosoftSemanticKernelVersion>1.22.0</MicrosoftSemanticKernelVersion>
     <MicrosoftSemanticKernelExperimentalVersion>1.22.0-alpha</MicrosoftSemanticKernelExperimentalVersion>
     <MicrosoftExtensionsAIVersion>9.0.0-preview.9.24525.1</MicrosoftExtensionsAIVersion>
+
+    <NuGetAudit>true</NuGetAudit>
```

Comment:
You need to set `NuGetAuditMode` to `all` as well.  And I think `NuGetAudit` is already enabled in .net 8 by default